pkg/minikube/assets/addons.go

```diff
@@ -277,6 +277,21 @@ var Addons = map[string]*Addon{
 			"istio-default-profile.yaml",
 			"0640"),
 	}, false, "istio", "3rd party (Istio)", "", "https://istio.io/latest/docs/setup/platform-setup/minikube/", nil, nil),
+	"inspektor-gadget": NewAddon([]*BinAsset{
+		MustBinAsset(addons.InspektorGadgetAssets, "inspektor-gadget/ig-namespace.yaml", vmpath.GuestAddonsDir, "ig-namespace.yaml", "0640"),
+		MustBinAsset(addons.InspektorGadgetAssets, "inspektor-gadget/ig-serviceaccount.yaml", vmpath.GuestAddonsDir, "ig-serviceaccount.yaml", "0640"),
+		MustBinAsset(addons.InspektorGadgetAssets, "inspektor-gadget/ig-role.yaml", vmpath.GuestAddonsDir, "ig-role.yaml", "0640"),
+		MustBinAsset(addons.InspektorGadgetAssets, "inspektor-gadget/ig-rolebinding.yaml", vmpath.GuestAddonsDir, "ig-rolebinding.yaml", "0640"),
+		MustBinAsset(addons.InspektorGadgetAssets, "inspektor-gadget/ig-clusterrole.yaml", vmpath.GuestAddonsDir, "ig-clusterrole.yaml", "0640"),
+		MustBinAsset(addons.InspektorGadgetAssets, "inspektor-gadget/ig-clusterrolebinding.yaml", vmpath.GuestAddonsDir, "ig-clusterrolebinding.yaml", "0640"),
+		MustBinAsset(addons.InspektorGadgetAssets, "inspektor-gadget/ig-crd.yaml", vmpath.GuestAddonsDir, "ig-crd.yaml", "0640"),
+		MustBinAsset(addons.InspektorGadgetAssets, "inspektor-gadget/ig-daemonset.yaml.tmpl", vmpath.GuestAddonsDir, "ig-daemonset.yaml", "0640"),
+	}, false, "inspektor-gadget", "3rd party (kinvolk.io)", "", "https://minikube.sigs.k8s.io/docs/handbook/addons/inspektor-gadget/",
```

Comment:
```suggestion
	}, false, "inspektor-gadget", "3rd party (kinvolk.io)", "<github_user_here>", "https://minikube.sigs.k8s.io/docs/handbook/addons/inspektor-gadget/",
```

Before adding new addons we would like to have a maintainer listed (GitHub username), this is someone we can reach out to if say the addon is going to break with a new version of Kubernetes or a critical CVE is found in the image as we have little knowledge about the addon itself. Could you please find someone from the project that's willing to take this responsibility and put their GitHub username in the above show spot. Us reaching out will likely be extremely rare so it wouldn't be much or a burden, thanks.

Reply:
An org list would work for sure, feel free to use the link you provided in place of the GitHub username.

Reply:
> Shall I add the link to the org people in the space for github user? `https://github.com/orgs/inspektor-gadget/people`

Yup, that will work